spec/rubocop/cop/style/empty_line_between_defs_spec.rb

```diff
@@ -141,6 +141,18 @@
                              '  def b; end'].join("\n"))
   end
 
+  it 'treats lines with whitespaces as blank' do
```

Comment:
Got it, then I think it would be better to support it just in case someone disables trailing whitespace cop.